src/redux/news/news.sagas.js

```diff
@@ -85,10 +73,7 @@ function* handleAddNews({ payload }) {
 function* handleNewsDelete({ payload }) {
   try {
     yield put(setNewsLoading(true));
-    const result = yield call(deleteArticle, payload);
-    if (result.message) {
-      yield call(handleCustomNewsError, result.message[errorsLanguage].value);
-    }
+    yield call(deleteArticle, payload);
```

Comment:
Why do we fetch again all the news after the deleting of the article? It is a useless HTTP call. Maybe, it is much better to just remove the article from the redux store by id. It will reduce the quantity of HTTP requests: 2 requests => 1 request